src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1935,7 +1938,9 @@ def export_template_spec(cmd, output_folder, resource_group_name=None, name=None
         version = id_parts.get('resource_name')
         if version == name:
             version = None
-    exported_template = rcf.template_spec_versions.get(resource_group_name, name, version) if version else rcf.template_specs.get(resource_group_name, name)
+    if not version:
+        raise IncorrectUsageError('a template spec version must be specified for export')
```

Comment:
Please use a more friendly voice
```suggestion
        raise IncorrectUsageError('Please specify the template spec version for export')
```